src/Microsoft.ML.Data/Transforms/ColumnSelecting.cs

```diff
@@ -33,8 +33,32 @@
 namespace Microsoft.ML.Transforms
 {
     /// <summary>
-    /// The ColumnSelectingEstimator supports selection of specified columns to keep from a given input.
+    /// Keeps or drops selected columns from an <see cref="IDataView"/>.
     /// </summary>
+    /// <remarks>
+    /// <format type="text/markdown"><![CDATA[
+    ///
+    /// ###  Estimator Characteristics
+    /// |  |  |
+    /// | -- | -- |
+    /// | Does this estimator need to look at the data to train its parameters? | No |
+    /// | Input columns data type | Any |
+    /// | Output column data type | When not dropped, the data type is unchanged |
```

Comment:
> | Output column data type | When not dropped, the data type is unchanged | [](start = 7, length = 75)

I don't think this is applicable, since we don't produce output columns.

Reply:
Ok you are right, let me remove that line.

---
In reply to: [277078096](https://github.com/dotnet/machinelearning/pull/3428#discussion_r277078096) [](ancestors = 277078096)